llvm/lib/DebugInfo/Symbolize/DIPrinter.cpp

```diff
@@ -309,6 +314,9 @@ static json::Object toJSON(const DILineInfo &LineInfo) {
        {"Line", LineInfo.Line},
        {"Column", LineInfo.Column},
        {"Discriminator", LineInfo.Discriminator}});
+  if (LineInfo.IsApproximatedLine)
+    Obj.insert({"Approximate", LineInfo.IsApproximatedLine});
```

Comment:
Obj.insert({"Approximate", true});

Reply:
IsApproximateLine is a boolean value. I am not opposed to hardcode the "true"  for Approximate. But ultimately this would have no effect in the desired outcome.

Reply:
Just my personal preference here - let's leave it as it is unless someone has a stronger reason.